engpetermwangi/Politico2
# [Chore #163682014] Unit tests for the '/parties' endpoint

####What does this PR do?
Tests the '/parties' endpoint of the Politico REST API for creation of political party given the necessary json and when not adequate data in json format is provided.

####Description of tasks to be completed?
Have the tests running for the '/parties' endpoint

####How can this be manually tested?
After cloning the repo, cd into the it and run python test_politico.py

####Any background context to be provided?
Make sure to have the flask and pytest (and the other libraries in requirements.txt) installed. The tests fail at first until the code is refactored to pass.

####Relevant Pivotal Tracker stories?
#163682014
